db/db_filesnapshot.cc

```diff
@@ -151,6 +151,33 @@ Status DBImpl::GetSortedWalFiles(VectorLogPtr& files) {
     }
   }
 
+  if (s.ok()) {
+    size_t wal_size = files.size();
+#ifndef NDEBUG
+    std::ostringstream wal_names;
+    for (const auto& wal : files) {
+      wal_names << wal->PathName() << " ";
+    }
+
+    std::ostringstream wal_required_by_manifest_names;
+    for (const auto& wal : required_by_manifest) {
+      wal_required_by_manifest_names << wal << ", ";
```

Comment:
nit: this will add `,` at the end not consistent with `wal_names`

Reply:
wal_required_by_manifest_names is number only so added ",". I can do "," for wal_names too.

Reply:
Fixed